src/sentry/api/serializers/models/group.py

```diff
@@ -250,10 +250,11 @@ def serialize(self, obj, attrs, user):
             permalink = None
 
         is_subscribed, subscription = attrs['subscription']
+        share_id = obj.get_share_id()
```

Comment:
should fetch this in bulk as part of get_attrs

Reply:
Oh, right, this operates on potentially multiple at a time. Yeah, I'll do that.

Reply:
just imagine a button in the UI -- it'd be silly to make two api requests to rotate

Reply:
Done. 89bbeda7006745322e7addfeafa50dff752186b4